### DISCUSSION TITLE
Filter Table

### DISCUSSION THREAD

**Discussion by emyr01** [rappasoft/laravel-livewire-tables#1236] [Q&A]
Currently im working on the filter, it only filtered by data in database, it is possible to get value from messages?  how can i fix this code? (ex: it only display add_roles, but i want the filter to display Add Roles

Here i attach my livewire table for filter, messages and my database 
![image](https://github.com/rappasoft/laravel-livewire-tables/assets/93184571/82fb41e4-4706-41ed-bc98-7e162b3c8218)
![image](https://github.com/rappasoft/laravel-livewire-tables/assets/93184571/b046451b-78ed-4544-bd20-03005febfbb7)
![image](https://github.com/rappasoft/laravel-livewire-tables/assets/135106525/6f31caf2-52bb-49a1-982a-8b5762a8ce58)

**Comment by lrljoe** [comment]
Use a format or label method on the column.

**Reply by lrljoe** [reply]
Did this fix your issue, or do you need further assistance?

**Reply by emyr01** [reply]
Yes can i have a further assistance, i already try the format method on the column like this but it still get the filtered value directly from the database not from the messages file, is there any other ways?

 
`Column::make(__('messages.display_name'), 'display_name')
                ->sortable()
                ->searchable()
                ->format(
                    fn($value, $row, $column) => __('messages.' . $row->display_name)
                ),`

**Reply by lrljoe** [reply]
Apologies for not coming back, I'm hoping you've fixed it now?

If not, make sure your Lang file is returning an array, and that you've got the language set in your app.config file.
